TESTS/mbedmicro-rtos-mbed/MemoryPool/main.cpp

```diff
@@ -0,0 +1,640 @@
+#include "mbed.h"
+#include "greentea-client/test_env.h"
+#include "unity.h"
+#include "utest.h"
+#include "rtos.h"
+#include "MemoryPool.h"
+
+using namespace utest::v1;
+
+
+/* Enum used to select block allocation method. */
+typedef enum{
+    ALLOC,
+    CALLOC
+} AllocType;
+
+/* Structure for complex block type. */
+typedef struct{
+    int a;
+    char b;
+    int c;
+}COMPLEX_TYPE;
+
+/* Function to check if complex type object is cleared.*/
+bool comp_is_cleared(COMPLEX_TYPE *object)
+{
+    if(object->a == 0 &&
+       object->b == 0 &&
+       object->c == 0)
+    {
```

Comment:
please align this file according to https://docs.mbed.com/docs/mbed-os-handbook/en/latest/cont/code_style/